packages/components/table/src/util.ts

```diff
@@ -115,6 +115,7 @@ export const getColumnByKey = function <T>(
   let column = null
   for (let i = 0; i < table.columns.length; i++) {
     const item = table.columns[i]
+    if (!item.columnKey) throw new Error('column does not have columnKey set')
```

Comment:
```suggestion
```

```diff
    for (let i = 0; i < table.columns.length; i++) {
      const item = table.columns[i]
-     if (!item.columnKey) throw new Error('column does not have columnKey set')
      if (item.columnKey === columnKey) {
        column = item
        break
      }
    }
+   if (!column) throw new Error(`[Table] No column matching with column-key: ${columnKey}`)
    return column
```

Reply:
good suggestion 👍 !  Maybe also use `throwError`

https://github.com/element-plus/element-plus/blob/742a37ca58aa4833653b3c0d91d5157610ce4463/packages/utils/error.ts#L10

Reply:
That's much better！I didn't notice it before.